ledger-core/virtual/execute/execute.go

```diff
@@ -67,6 +68,7 @@ type SMExecute struct {
 	messageSender         messageSenderAdapter.MessageSender
 	pulseSlot             *conveyor.PulseSlot
 	authenticationService authentication.Service
+	globalSemaphore       tool.RunnerLimiter
```

Comment:
maybe call it executionLimiter or something like that? semaphore with type that is not semaphore, when we have semaphore as different type.

Reply:
see conversation above, renamed it after @bigbes  request